### ISSUE TITLE
Redis ACL (user/pass) support?

### ISSUE THREAD

**Issue by LangJV** [OpenIDC/mod_oauth2#63]
Is it currently possible to utilize a redis username AND password when authenticating to Redis?

The documentation mentions the ability to pass a password, but quite a few years back REdis becan supporting true ACLs, meaning you can now authenticate with a username AND a password.

I'd like to know if i pass like: "OAuth2username" or "username" or "OAuth2redisusername" - will it be able to authenticate with a username and password?

**Comment by zandbelt** [comment]
yes, see: https://github.com/OpenIDC/mod_auth_openidc/blob/v2.4.15.3/auth_openidc.conf#L686-L689

**Comment by LangJV** [comment]
As a follow-up question (and apoloiges if this is super basic) - how does one provide any of the redis arguments:
Looking at: https://github.com/OpenIDC/mod_oauth2/blob/aa178d4bf115207a28517616919b0abe7a796388/oauth2.conf#L27-L57

would i do something like:
OAuth2Cache redis host myhost port 1234 password mypassword

Or:
OAuth2Cache redis
OAuth2Cache host myhost
OAuth2Cache port 1234
OAuth2Cache password mypassword

Or:
OAuth2Cache redis host myhost
OAuth2Cache redis port 1234
OAuth2Cache redis password mypassword

I've tried googling but the internet doesnt seem to have examples either. I see the doc defines the syntax as: #OAuth2Cache <type> [<options>]
But that seems a bit ambiguous (maybe just to me?)

**Comment by zandbelt** [comment]
it is a query-encoded format, as documented - with samples - in:
https://github.com/OpenIDC/mod_oauth2/blob/aa178d4bf115207a28517616919b0abe7a796388/oauth2.conf#L68-L73
I notice that since the order of the statements in the sample changed (OAuth2Cache needs to be specified before others), this is less than obvious now, I'll improve that
so `OAuth2Cache host=myhost&port=1234&password=mypassword`
there are also a few examples in the README.md which is probably what most people go with
